src/frontend/src/instance/prom_store.rs

```diff
@@ -279,6 +312,15 @@ impl PromStoreProtocolHandler for ExportMetricHandler {
         Ok(())
     }
 
+    async fn write_fast(
+        &self,
+        _request: RowInsertRequests,
+        _ctx: QueryContextRef,
+        _with_metric_engine: bool,
+    ) -> ServerResult<()> {
+        unimplemented!()
```

Comment:
Returns an error instead of panic?

Reply:
> Returns an error instead of panic?

This follows the other method implementation:

https://github.com/GreptimeTeam/greptimedb/blob/4acf27a07697166683c3144899f98c3dd3f7457d/src/frontend/src/instance/prom_store.rs#L324-L330